docs/running_e2etests_locally.md

```diff
@@ -0,0 +1,25 @@
+# Running e2e tests locally
+
+Build the application, which means compiling the Go code, building the docker image and the helm chart package.
+
+```bash
+CGO_ENABLED=0 go build -a -v -ldflags "-w -linkmode 'auto' -extldflags '-static' -X 'main.gitCommit=`git rev-parse HEAD`'" \
```

Comment:
It would be cool to have an `architect` command to run these commands, something like `architect build --docker --chart`

Reply:
what do you think @giantswarm/wg-integration-testing ?

Reply:
Using a binary makes sense to me.

I just get confused what should be in architect vs devctl. But this is using architect helm template so maybe its the right place.

But I'd like to hear others opinion.

Reply:
I figured `architect` would be best since it contained `architect build` and `architect helm template`. But open to suggestions.

Reply:
OK cool that makes sense to me.